content/collections/experiment/en/experiment-quick-start.md

```diff
@@ -0,0 +1,266 @@
+
+* **Product experimentation**: running experiments and A/B tests to onboard new users, reduce friction for checkout experiences, roll out new features, and more.
+* **Progressive feature delivery:** Pre-plan and stage new features for beta testers, a percentage of your users, or even specific target audiences.
+* **Dynamic in-product experiences:** Deploy and adapt custom experiences at scale.
+
+Amplitude Experiment enables experimentation through feature flags. Feature flags are switches that let you modify your product's experience without having to change code. Experiment and feature flags use the [Amplitude Experiment SDK](/docs/sdks/experiment-sdks) or [REST API](/docs/apis/experiment) to communicate with Amplitude Experiment.
+
+Setting up an experiment is a multi-stage process encompassing the following procedures:
+1. Installing and setting up SDKs and deployments
+2. Creating feature flags
+3. Selecting the deployment and creating a payload
+4. Creating variations and sending payloads
+5. Designing experiments
+
+{{partial:admonition type="warning" heading=""}}
+This quick start guide contains only the basic information needed to implement Experiment. Review the entire set of [Experiment documentation](docs/feature-experiment) to understand the full complexity and features of the product.
+{{/partial:admonition}}
+
+## Prerequisites
+Before you can begin using experiments, you must first:
+
+- Have access to the [Experiment SDK](/docs/sdks/experiment-sdks).
+- Make sure you have access to the Experiment feature. 
+- Ensure you have developer access to the application where you'll integrate your feature flags.
+
+## Setting up the SDK
+
+Install the Amplitude SDK with the Experiment client. For example, through javascript:
+
+```bash
+
+npm install @amplitude/analytics-browser @amplitude/experiment-browser
+
+import * as amplitude from '@amplitude/analytics-browser'; 
+import { Experiment } from '@amplitude/experiment-browser'; 
+amplitude.init('AMPLITUDE_API_KEY');  
+const experiment = Experiment.initialize('EXPERIMENT_API_KEY'); 
+await experiment.start();
+```
+
+## Setting up a deployment
+
+Experiment uses the same projects as Amplitude Analytics. As a best practice, create one project for each product and each environment. Because [flags](#flags-and-experiments), [experiments](#flags-and-experiments), and [deployments](#deployments) only exist within a single project, you must duplicate these objects across projects within the same product.
+
+In Amplitude Experiment, a deployment serves a group of flags or experiments for use in an application. Each [project](#projects) has a deployment using the project API key as the deployment key, available by default. On creation, experiment deployments are assigned a randomly generated deployment key which Experiment uses to identify the deployment and authorize requests to the evaluation servers.
```

Comment:
I think this paragraph might be wrong.

Projects don't have deployments by default. You need to create the deployment and assign it to a project.